app/scripts/controllers/bridge/bridge-controller.ts

```diff
@@ -319,4 +340,25 @@ export default class BridgeController extends StaticIntervalPollingController<
       'NetworkController:getSelectedNetworkClient',
     );
   }
+
+  /**
+   *
+   * @param contractAddress - The address of the ERC20 token contract
+   * @param walletAddress - The address of the wallet
+   * @param chainId - The hex chain ID of the bridge network
+   * @returns The atomic allowance of the ERC20 token contract
+   */
+  getBridgeERC20Allowance = async (
+    contractAddress: string,
+    walletAddress: string,
+    chainId: Hex,
```

Comment:
We can retrieve the wallet address using

```
const { address: walletAddress } = this.#getSelectedAccount();
```

Reply:
Done in [961dddc](https://github.com/MetaMask/metamask-extension/pull/27262/commits/961dddcd4d6a4716b2f438aba274075cdaf33099)